src/crypto/utils/setup.py

```diff
@@ -0,0 +1,6 @@
+from collections import namedtuple
+
+DayDim = namedtuple(typename="DayDim", field_names="day_key name")
```

Comment:
Lately I always defined those with typing, easy to do FYI: https://www.youtube.com/watch?v=ljQHjQx76Po

Reply:
Love this. I wasn't aware. Thank you.